docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,18 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+\section{Overflow\label{value-checker-overflow}}
+\begin{sloppypar}
```

Comment:
I don't see the point of the sloppypar.  The manual formats identically with and without it.